src/all/pandachaika/src/eu/kanade/tachiyomi/extension/all/pandachaika/PandaChaikaDto.kt

```diff
@@ -54,7 +54,7 @@ class LongArchive(
         val characters = filterTags("character", tags = tags)
         val male = filterTags("male", tags = tags)
         val female = filterTags("female", tags = tags)
-        val others = filterTags(exclude = listOf("female", "male", "artist", "publisher", "group", "parody"), tags = tags)
+        val others = filterTags(exclude = listOf("language", "female", "male", "artist", "publisher", "group", "parody"), tags = tags)
```

Comment:
not sure if this will work... as `filterTags` function above have `include` default to empty string, which makes the filter ":"